src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -19,6 +27,25 @@ def post(self, request, sentry_app):
         if sentry_app.is_publish_request_inprogress:
             return Response({"detail": "Publish request in progress."}, status=400)
 
+        if features.has("organizations:sentry-app-logo-upload", sentry_app.owner):
+            if not SentryAppAvatar.objects.filter(
+                sentry_app=sentry_app, color=True, avatar_type=1
```

Comment:
The `1` in the `avatar_type=1` filter is difficult to understand in this file. Can you create an enum:
```py
class AvatarTypes(enum.Enum):
    LETTER_AVATAR = 0
    UPLOAD = 1

    @classmethod
    def get_choices(cls) -> Sequence[tuple[int, str]]:
        return tuple((_.value, _.name.lower()) for _ in AvatarTypes)
```